x-pack/test/functional/config.js

```diff
@@ -195,8 +197,11 @@ export default async function ({ readConfigFile }) {
       },
       transform: {
         pathname: '/app/kibana/',
-        hash: '/management/elasticsearch/transform'
-      }
+        hash: '/management/elasticsearch/transform',
+      },
+      endpoint: {
+        pathname: '/app/endpoint',
```

Comment:
is this right?

Reply:
@oatkiller this looks right. I'm testing it in my app now.